src/common/user_intf.f90

```
@@ -222,6 +238,13 @@ end subroutine user_intf_init
   ! when running in pure turboNEKO mode
   !
 
+  !> Dummy user (fluid) implicit Brinkman forcing
+  subroutine dummy_implicit_brinkman(chi, t)
+    type(field_t), intent(inout) :: chi
+    real(kind=rp), intent(in) :: t
+    chi%x = 0.0
+  end subroutine dummy_implicit_brinkman
+
```

This should throw an error, as all the other dummies.

---

Not all dummies I guess, some are "do nothing" but will not return an error. Like dummy_user_check